README.md

```diff
@@ -211,33 +211,18 @@ To run the tests against a remote deployment you will need to specify the enviro
 
 For deploying with Concourse see the [CI README](./ci/README.md).
 
-### Deployment with [Helm](https://helm.sh/)
```

Comment:
As discussed, we should also document how to deploy directly via `gcloud`.

Reply:
Added new `deployment with gcloud` section, thanks!